server/routes/cohorts.js

```diff
@@ -57,4 +62,63 @@ router.get(
   })
 );
 
+// Assign participant
+router.post(
+  '/:id/assign/:participantId',
+  [applyMiddleware(keycloak.allowRolesMiddleware('health_authority', 'employer'))],
+  asyncMiddleware(async (req, res) => {
+    const { user_id: userId, sub: localUserId } = req.user;
+    const user = userId || localUserId;
+    const { id, participantId } = req.params;
+    if (id && participantId) {
+      const [participant] = await getParticipantByID({ id: +participantId });
```

Comment:
what does the `+` operator do here? I see you using it on line 78 as well.

Reply:
convert number to string